client/src/index.css

```
@@ -4,68 +4,129 @@
     --chart-4: 280 65% 60%;
     --chart-5: 340 75% 55%;
+    --sidebar: 240 4% 10%;
+    --sidebar-foreground: 0 0% 98%;
+    --sidebar-muted: 240 5% 78%;
+  }
+  * {
+    @apply border-border;
+  }
+  html {
+    font-size: 16px;
+    text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)